### PR TITLE
Feature/aut 4908 standalone authorizer

### PR DESCRIPTION
**Pull request by jkotiuk**
## Description

Add helm chart for standalone authorizer.

## Type of changes

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Tests (extending the test suite)
- [ ] Refactor (internal improvement that doesn't change product functionality)
- [ ] Other (if none of the other choices apply)

### CODE REVIEW

File: charts/standalone-authorizer/templates/deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+              value: "{{ .Values.issuerURL }}"
+            - name: ACP_TENANT_ID
+              value: "{{ .Values.tenantID }}"
+            - name: ACP_SERVER_ID
+              value: "{{ .Values.serverID }}"
+            - name: HTTP_SERVER_PORT
+              value: "{{ .Values.service.port }}"
+            - name: HTTP_SERVER_DANGEROUS_DISABLE_TLS
+              value: "{{ .Values.tlsDisabled }}"
+          {{- range $key, $value := .Values.extraEnv }}
```
Comment (ppodbielski-cloudentity) [inline charts/standalone-authorizer/templates/deployment.yaml:57]: @holowinski do we want to use env? as I remember you mention that we want to use configmap only

File: charts/standalone-authorizer/values.yaml [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+
+## Vanity Domain
+## If the issuerURL uses a vanity domain, you will need to specify the tenantID,
+## and if the domain is per-server, then you must also specify the terverID.
+## There is no harm in specifying them, even if you do not use a vanity domain.
+tenantID: "default"
+serverID: "system"
+
+## Credentials for your ACP standalone-authorizer
+##
```
Comment (ppodbielski-cloudentity) [inline charts/standalone-authorizer/values.yaml:56]: ## double

File: charts/standalone-authorizer/values.yaml [hunk 3]
```diff
@@ -0,0 +1,197 @@
...
+## If you're using multi-tenant standalone-authorizer this should point
+## to the `system` workspace of your `system` tenant.
+## i.e. "http://yourdomain.com/system/system"
+##
+issuerURL: "https://acp.acp-system:8443/default/system"
+
+## Vanity Domain
+## If the issuerURL uses a vanity domain, you will need to specify the tenantID,
+## and if the domain is per-server, then you must also specify the terverID.
+## There is no harm in specifying them, even if you do not use a vanity domain.
```
[the remark this answers is not in the document]
Comment (sburke-ce) [reply charts/standalone-authorizer/values.yaml:51]: `terverID` -> `serverID`.  Are there any other concerns?

**Review by sburke-ce — APPROVED** [review]

File: charts/standalone-authorizer/values.yaml [hunk 4]
```diff
@@ -0,0 +1,207 @@
...
+##
+extraEnv: {}
+
+## node selector for pod assignment
+##
+## nodeSelector: {}
+
+## Affinity for pod assignment
+##
+affinity: {}
```
Comment (ppodbielski-cloudentity) [inline charts/standalone-authorizer/values.yaml:93]: acp:
```  ## Affinity for pod assignment
  affinity: {}
  #   nodeAffinity:
  #     requiredDuringSchedulingIgnoredDuringExecution:
  #       nodeSelectorTerms:
  #         - matchExpressions:
  #             - key: fission
  #               operator: In
  #              values:
  #                 - "true"
  ```

**Review by mbilski — CHANGES_REQUESTED** [review]
This helm chart should match istio-authorizer. There were lots of changes there recently

**Pull request closed without merging**